model/Cargo.toml

```diff
@@ -14,7 +14,7 @@ primitive-types = { version = "0.8" }
 secp256k1 = "0.20"
 serde = { version = "1.0", features = ["derive"] }
 serde_with = { version = "1.6", default-features = false, features = ["macros"] }
-web3 = { version = "0.15", default-features = false }
+web3 = { version = "0.15", default-features = false, features = ["signing"] }
```

Comment:
What are we using  from the signing feature that we didn't use before?

Reply:
The issue here is that when the entire project is compiled together, everything was fine but when you try to compile/build/run/test `model` stand alone with something like:

```sh
cargo build -p model --tests
```
then we run into the following errors:

```
➜  gp-v2-services git:(320/cancellation_implementation) ✗ cargo build -p model --tests
   Compiling model v0.1.0 (/Users/bensniff/Projects/gnosis/gp-v2-services/model)
error[E0432]: unresolved imports `web3::signing::Key`, `web3::signing::SecretKeyRef`
  --> model/src/order.rs:18:27
   |
18 | use web3::signing::{self, Key, SecretKeyRef};
   |                           ^^^  ^^^^^^^^^^^^ no `SecretKeyRef` in `signing`
   |                           |
   |                           no `Key` in `signing`

error[E0425]: cannot find function `recover` in module `signing`
  --> model/src/lib.rs:89:18
   |
89 |         signing::recover(&message, &signature, recovery_id).ok()
   |                  ^^^^^^^ not found in `signing`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0425, E0432.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `model`
```

More concisely, we import it here and it is used several times in this file.

http://github.com/gnosis/gp-v2-services/blob/456859c8a1c0a76f27f986e5296e8602a4fdff63/model/src/order.rs#L18-L18